### PR TITLE
Implement <amp-ad type="adform"> support

### PR DESCRIPTION
**Pull request by simast**
This PR adds Adform ad support to `<amp-ad>` tag.

### CODE REVIEW

File: ads/adform.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    let lastError;
+
+    // NOTE: Since validateSrcPrefix() throws and we need to validate src to be
+    // from one of the two whitelisted hosts - we will look for two exceptions
+    // and re-throw validation error with invalid "data-src" attribute.
+    for (const type in hosts) {
+
+      try {
+        validateSrcPrefix(hosts[type], src);
```
Comment (cramforce) [inline ads/adform.js]: Would you mind instead changing `validateSrcPrefix` to take an array of prefixes? And add some unit tests there?

File: ads/adform.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Adform
+
```
Comment (cramforce) [inline ads/adform.md]: Could you add a link to a place where people can find additional documentation?

**Comment by cramforce** [comment]
This looks excellent! Thanks! I added a few comments.

**Comment by cramforce** [comment]
Please rebase. I changed the `amp-ad` docs around a bit.

**Comment by simast** [comment]
@cramforce: Squashed and updated per your comments!

**Comment by cramforce** [comment]
Excellent! LGTM

**Pull request merged**